src/scenarios/blazorlocalized/src/BlazorLocalized/BlazorLocalized.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
```

Comment:
Could you explain how this project was created? And what is additional/changed from the template that you used.
And add that to the PR description too.

Reply:
> What exactly, beyond the wrong default net version, looks wrong?

It isn't clear which parts of the project are the original template, and which ones were changed/added and need to be reviewed.

And you wrote the library project?

Reply:
I see. I updated the description, it should be easier now.